kafka-ui-react-app/src/components/Dashboard/ClustersWidget/__test__/ClustersWidgetContainer.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { containerRendersView } from 'lib/testHelpers';
+import ClustersWidget from 'components/Dashboard/ClustersWidget/ClustersWidget';
+import ClustersWidgetContainer from 'components/Dashboard/ClustersWidget/ClustersWidgetContainer';
+
+describe('ClustersWidgetContainer', () => {
+  containerRendersView(<ClustersWidgetContainer />, ClustersWidget);
+  describe('view empty ClusterWidget', () => {
+    const setupEmptyWrapper = () => (
+      <ClustersWidget clusters={[]} onlineClusters={[]} offlineClusters={[]} />
+    );
+    it(' is empty when no online clusters', () => {
+      const wrapper = mount(setupEmptyWrapper());
+      expect(wrapper.find('.is-success').text()).toBe('0');
+    });
+    it(' is empty when no offline clusters', () => {
+      const wrapper = mount(setupEmptyWrapper());
+      expect(wrapper.find('.is-danger').text()).toBe('0');
+    });
+  });
```

Comment:
there is no need to test <ClustersWidget> here again

Reply:
deleted